src/app/home.component.ts

```diff
@@ -83,6 +90,16 @@ export class HomeComponent implements OnInit, OnDestroy {
       // thus, refresh the scenario list
       this.contextService.refresh();
     });
+
+    this.accesscode = this.route.snapshot.queryParams['ac'];
+    if( this.accesscode ){
+      this.accessCodeLinkSuccessAlert = `${this.accesscode} added`;
+      this.accessCodeLinkSuccessClosed = false;
+      this.location.go("/app/home");
+      setTimeout(() => {
```

Comment:
Why are you reloading the page?
Is it not possible to just remove the accessCodeLinkSuccessAlert again or set it to closed?

Reply:
It's done. I set it to closed, but I nested inside setTimeout 5 second delay